projects/wave-core/src/lib/operators/dialogs/data-repository/data-repository.component.html

```diff
@@ -18,13 +18,12 @@
     </mat-menu>
     -->
 </wave-sidenav-search>
-
 <mat-list *ngIf="(sources | async).length > 0; else noSources">
     <ng-template ngFor let-source [ngForOf]="sources | async | waveMappingDataSourceFilter:searchTerm">
-        <wave-vector-source-dataset *ngIf="source.operator === 'ogr_source'; else rasterDataset" [dataset]="source">
+        <wave-vector-source-dataset *ngIf="source.operator === 'ogr_source'; else rasterDataset" [dataset]="source" [searchTerm]=searchTerm>
```

Comment:
```suggestion
        <wave-vector-source-dataset *ngIf="source.operator === 'ogr_source'; else rasterDataset" [dataset]="source" [searchTerm]="searchTerm">
```

Although it seems to work, you should use quotation marks around these properties.

Reply:
added question marks